src/views/VisitsView.vue

```diff
@@ -167,6 +181,9 @@ export default {
   components: { QueryRecordings, EventSummary },
   data() {
     return {
+      showInfo: this.isInfoShown(),
+      infoMessage: `Visits are multiple recordings and tracks combined into a likely visit of a single predator/animal. 
+      Each visit can be expanded by clicking the header to show the events ( a track of a recording) which make up this visit.`,
```

Comment:
Suggested edit:

A "visit" is multiple thermal video tracks that have been combined because they are likely to be due to the appearance of a single animal. Each visit can be expanded by clicking on it to show the tracks which it is made up from.